### PR TITLE
docs(ingest/kafka-connect): add details on platform instance mapping

### PR DESCRIPTION
**Pull request by mayurinehate**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/docs/sources/kafka-connect/kafka-connect.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
 ## Advanced Configurations
 
+### Working with Platform Instances
+If you've multiple instances of kafka OR source/sink systems that are referred in your kafka-connect, you'd need to configure platform instance for these systems in kafka-connect recipe to generate correct lineage edges. You must have already set `platform_instance` in recipes of original source/sink systems. Refer the document [Working with Platform Instances](https://datahubproject.io/docs/platform-instances) to understand more about this.
```
Comment (anshbansal) [inline metadata-ingestion/docs/sources/kafka-connect/kafka-connect.md:4]: Let's please change kafka-connect -> `kafka-connect` to make this easier to read.

**Review by anshbansal — APPROVED** [review]
1 small change requested. Other than that LGTM

File: metadata-ingestion/docs/sources/kafka-connect/kafka-connect.md (hunk 1 above)
Comment (mayurinehate) [reply metadata-ingestion/docs/sources/kafka-connect/kafka-connect.md]: done

**Comment by anshbansal** [comment]
This is docs only PR and docs build is passing so merging this in

**Pull request merged**